src/chess/uciloop.cc

```diff
@@ -153,13 +154,18 @@ bool UciLoop::DispatchCommand(
   } else if (command == "ucinewgame") {
     CmdUciNewGame();
   } else if (command == "position") {
-    if (ContainsKey(params, "fen") == ContainsKey(params, "startpos")) {
-      throw Exception("Position requires either fen or startpos");
-    }
-    const std::vector<std::string> moves =
-        StrSplitAtWhitespace(GetOrEmpty(params, "moves"));
-    CmdPosition(GetOrEmpty(params, "fen"), moves);
-  } else if (command == "go") {
+	  if (ContainsKey(params, "fen") == ContainsKey(params, "startpos")) {
+		  throw Exception("Position requires either fen or startpos");
+	  }
+	  const std::vector<std::string> moves =
+		  StrSplitAtWhitespace(GetOrEmpty(params, "moves"));
+	  CmdPosition(GetOrEmpty(params, "fen"), moves);
+  } else if (command == "get") {
```

Comment:
There's no uci command "get" (I've just checked stockfish doesn't have it either).
If stockfish supports it in some way, it's better to be consistent.
If not, maybe we don't need it either.

In any case, I don't think that adding non-standard commands in `UciLoop` is a good idea.
Instead, we should probably add `virtual bool CmdNonStandardCommand(const std::string& command);` which uci loop should call in such cases, and `EngineLoop::CmdNonStandardCommand` should just have `if (command == "blah") DoThings(); `

Reply:
There is no uci command indeed. Now that lc0 is champion we could set our own standards.

Just kidding ;) 
Introduced a CmdNonStandardUciCommand method and moved code up to EngineLoop. However, there is a command-parser in UciLoop which I can't circumvent easily, so "get","fen" is still there...

Reply:
Found a way to move out 'get fen' from the uci commands. Having the sole 'line string' loses the parsing of command and tokens, but I guess it's good for now.